straxen/plugins/acqmon_processing.py

```diff
@@ -77,56 +76,57 @@ def infer_dtype(self):
         aq_ch_start = straxen.n_hard_aqmon_start
-        aqmon_channel_names = ('sum_wf','m_veto_sync',
-                               'hev_stop', 'hev_start', 'he_stop','he_start','busy_stop', 'busy_start')
+        aqmon_channel_names = ('sum_wf', 'm_veto_sync',
+                               'hev_stop', 'hev_start', 'he_stop', 'he_start', 'busy_stop', 'busy_start')
         channel_numbers = np.arange(aq_ch_start, aq_ch_start + len(aqmon_channel_names))
-        
+
         # Keeping ch_range as class attribute for now, maybe will need it for other features
         self.channel_range = self.config['channel_map']['aqmon']
         self.channel_map = dict(zip(aqmon_channel_names, channel_numbers))
-        self.veto_names = ['busy_','he_','hev_']
-    
+        self.veto_names = ['busy_', 'he_', 'hev_']
+
     def get_window_size(self):
         # Give a very wide window
-        return (self.config['max_veto_gap']*100)
-    
-    def compute(self, aqmon_hits): 
+        return (self.config['max_veto_gap'] * 100)
+
+    def compute(self, aqmon_hits):
         hits = aqmon_hits
-        
+
         # Make a dict to populate later
         res_temp = dict()
-        
+
         for i, veto in enumerate(self.veto_names):
             veto_hits = channel_select(hits, self.channel_map[veto + 'stop'], self.channel_map[veto + 'start'])
-            vetos = self.merge_vetos(veto_hits, gap = self.config['max_veto_gap'],\
-                                           dtype = strax.time_fields, t = 0)
-            
+            vetos = self.merge_vetos(veto_hits, gap=self.config['min_veto_gap'], \
+                                     dtype=strax.time_fields, t=0)
+
             # Check that we found a veto interval and update the resulting dict
             if len(vetos):
-                res_temp.setdefault("time",[]).extend(vetos['time'])
-                res_temp.setdefault("endtime",[]).extend(vetos['endtime'])
-                res_temp.setdefault("veto_interval",[]).extend((vetos['endtime'] - vetos['time']))
-                res_temp.setdefault("veto_type",[]).extend([veto + 'veto']*len(vetos))
-        
+                res_temp.setdefault("time", []).extend(vetos['time'])
+                res_temp.setdefault("endtime", []).extend(vetos['endtime'])
+                res_temp.setdefault("veto_interval", []).extend((vetos['endtime'] - vetos['time']))
+                res_temp.setdefault("veto_type", []).extend([veto + 'veto'] * len(vetos))
+
         res = strax.dict_to_rec(res_temp, dtype=self.dtype)
-        res.sort(order = 'time') 
-        
+        res.sort(order='time')
+
         return res
-    
+
     @staticmethod
     def merge_vetos(channels, gap, dtype, t):
         if len(channels):
-            start, stop = strax.find_peak_groups(channels, gap_threshold = gap)
-            result = np.zeros(len(start),dtype=dtype)
-            result['time'] = start 
+            start, stop = strax.find_peak_groups(channels, gap_threshold=gap)
```

Comment:
@AlexElykov this was the culprit

Reply:
Yeah I 've seen it from the bootstrax error message in the run json, but had no time to look at it proper. I will check this update later today on some benchmark runs and see maybe I can spot some more bugs in my code.